lti_provider/auth.py

```diff
@@ -70,6 +71,6 @@ def authenticate(self, request, lti):
 
     def get_user(self, user_id):
         try:
-            return User.objects.get(pk=user_id)
+            return get_user_model().objects.get(pk=user_id)
         except User.DoesNotExist:
```

Comment:
The exception here should catch `get_user_model().DoesNotExist` rather than User. The `User` import can then be removed from at Ln. 4

Reply:
Yeah.. there is too much going on in this `try` block. That could be re-organized a bit.